src/science/canopy/cable_canopy.F90

```diff
@@ -84,7 +84,7 @@ SUBROUTINE define_canopy(bal,rad,rough,air,met,dels,ssnow,soil,veg, canopy,clima
     TYPE (veg_parameter_type), INTENT(INOUT)    :: veg
 
 REAL :: reducedLAIdue2snow(mp)
-LOGICAL :: sunlit_veg_mask(mp) 
+logical :: sunlit_veg_mask(mp) 
```

Comment:
Don't the coding standards indicate we should be using capitals?  i.e. `LOGICAL` not `logical`